libraries/tokens/package.json

```diff
@@ -25,9 +25,9 @@
     "commonjs/*"
   ],
   "scripts": {
-    "prepare": "npm run build",
```

Comment:
missing `"prepare": "pnpm run build"` by the looks of it?

Reply:
*sigh* I’m fed up with this task, I need more coffee too…